examples/examples/fmt/fmt.rs

```diff
@@ -1,20 +1,20 @@
 #![deny(rust_2018_idioms)]
-use tracing::debug;
+use std::io;
+use tracing::{debug, Level};
 
 mod yak_shave;
 
 fn main() {
     let subscriber = tracing_subscriber::fmt::Subscriber::new();
+    tracing::subscriber::set_global_default(subscriber).expect("Could not set global default");
```

Comment:
a bunch of unrelated changes in this example, which we can probably back out.

Reply:
(I'm fine with changing the example to use `set_global_default` but I'd prefer it not to be associated with the commit that adds the registry)